frontend_tests/node_tests/popover_menus_data.js

```diff
@@ -36,14 +35,41 @@ const mike = {
     email: "mike@example.com",
     is_admin: false,
     is_guest: false,
+    is_member: true,
 };
 
 const bob = {
     user_id: 2000,
     full_name: "Test Bob",
     email: "bob@example.com",
+    is_admin: true,
+    is_guest: false,
+};
+
+const alice = {
+    user_id: 3000,
+    full_name: "Test Alice",
+    email: "alice@example.com",
+    is_admin: false,
+    is_guest: true,
+};
+
+const john = {
+    user_id: 4000,
+    full_name: "Test John",
+    email: "john@example.com",
+    is_admin: false,
+    is_guest: false,
+    is_spectator: true,
+};
+
+const harvey = {
+    user_id: 2000,
+    full_name: "Test Harvey",
+    email: "harvey@example.com",
     is_admin: false,
     is_guest: false,
+    is_moderator: true,
 };
```

Comment:
Let's rename these to moderator_user, admin_user, etc... and remove `john` since I don't think we should have a spectator user since spectators don't exist as users in database and cannot send any messages, it just a state of the UI.